src/js/core/services/ui-grid-util.js

```diff
@@ -1225,7 +1225,7 @@ module.service('gridUtil', ['$log', '$window', '$document', '$http', '$templateC
     function runFunc(endDate){
       lastCall = +new Date();
       func.apply(context, args);
-      $interval(function(){ queued = null; }, 0, 1);
+      $interval(function(){ queued = null; }, 0, 1, false);
```

Comment:
I'm not sure either.  I got this code on the net and it seemed to work ok.  The false flag is needed to prevent unnecessary digest